x-pack/plugins/cases/server/services/cases/transform.ts

```diff
@@ -192,6 +192,11 @@ export function transformSavedObjectToExternalModel(
   const status =
     STATUS_ESMODEL_TO_EXTERNAL[caseSavedObject.attributes?.status] ?? CaseStatuses.open;
 
+  // @ts-ignore
```

Comment:
@jonathan-buttner How do you get this type of error? I am not able to reproduce it. I remember discussing this with @adcoelho and it seems that `myObject[undefined] === undefined` which will fallback to the expression after the `??`.

Reply:
> I remember discussing this with @adcoelho and it seems that myObject[undefined] === undefined which will fallback to the expression after the ??.

Yup that was it.

So if like you said @jonathan-buttner 

> the attributes field will be undefined. For the status and severity I think this might cause an indexing error:

and we do `SEVERITY_ES_MODEL_TO_EXTERNAL[undefined]` the value returned should be `CaseSeverity.LOW`.

Reply:
> @jonathan-buttner How do you get this type of error? I am not able to reproduce it. I remember discussing this with @adcoelho and it seems that `myObject[undefined] === undefined` which will fallback to the expression after the `??`.
Ah I didn't realize that. We would get an indexing error if we typed the `caseSavedObject` to have an optional `attributes` field which is technically possible if a bulk response encounters an error for one of the ids. It doesn't need to be in this PR but I think we should change the type eventually otherwise it looks unnecessary as to why we have those `?` and it'd be easy to forget one in the future.